scripts/merge-pr.py

```diff
@@ -257,8 +299,13 @@ def fix_version_from_branch(branch, versions):
 print("\n=== Pull Request #%s ===" % pr_num)
 print("title\t%s\nsource\t%s\ntarget\t%s\nurl\t%s"
       % (title, pr_repo_desc, target_ref, url))
-continue_maybe("Proceed with merging pull request #%s?" % pr_num)
+continue_maybe("Proceed with updating or merging pull request #%s?" % pr_num)
 
 merged_refs = [target_ref]
 
-merge_hash = merge_pr(pr_num, target_ref)
+update = continue_maybe2("Update PR and push to github (y) or merge locally (n)?")
```

Comment:
let's make this explicity, maybe ``r`` (remote) or ``l`` (local) or ``q/n`` to do nothing.